unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/AnalyticsHelper/AnalyticsHelper.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using System.Collections.Generic;
+using DCL;
+using DCL.Controllers;
+using UnityEngine;
+
+public static class AnalyticsHelper
+{
+    private static IParcelScene GetSceneWherePlayerIsStanding()
```

Comment:
This can be already retrieved by using:
`CommonScriptableObjects.sceneID`

Also, it can be retrieved by using environment:
`Environment.i.world.state.currentSceneId`

And then doing:
`Envionment.i.world.state.loadedScenes[currentSceneId]`

Reply:
I like more the Environment approach! Changed, thank you!